diff --git a/handlers/views_handler_field_links.inc b/handlers/views_handler_field_links.inc
index 51fe6e6..7e32550 100644
--- a/handlers/views_handler_field_links.inc
+++ b/handlers/views_handler_field_links.inc
@@ -64,6 +64,7 @@ class views_handler_field_links extends views_handler_field {
   public function options_submit(&$form, &$form_state) {
     // Remove unselected options.
     $form_state['values']['options']['fields'] = array_filter($form_state['values']['options']['fields']);
+    parent::options_submit($form, $form_state);
   }
 
   /**
